jaseci_core/jaseci/svc/meta.py

```diff
@@ -23,10 +24,26 @@ def __init__(self, run_svcs=True):
     ###################################################
 
     def run(self, hook=None):
-        self.app = ApplicationContext()
+        self.app = JsOrc(self)
         self.populate_context()
         self.populate_services()
 
+    def post_run(self, hook=None):
+        if self.run_svcs:
+            self.app.build()
+            if self.is_automated():
+                logger.info("JsOrc is automated. Pushing interval check alarm...")
+                self.push_interval(1)
+            else:
+                logger.info("JsOrc is not automated.")
+
+    def push_interval(self, interval):
+        if self.running_interval == 0:
```

Comment:
I am still not sure I understand the purpose this `running_interval` serves. Let's discuss.